README.md

```diff
@@ -222,7 +220,7 @@ $ npm run test
 You can perform a project **deep cleanup** with:
 
 ```bash
-$ npm run cleanup
+$ yarn run cleanup
```

Comment:
Is `run` needed here? If yes, shouldn't we add it [here](https://github.com/skroutz/analytics.js/pull/31/commits/aed0fcefc2883150cf7ccf8bbf3479c2b4382f54#diff-04c6e90faac2675aa89e2176d2eec7d8R193), too?